packages/contentful--cra-template-create-contentful-app/template/README.md

```diff
@@ -45,6 +45,33 @@ To make your app look and feel like Contentful use the following libraries:
 - [Forma 36](https://f36.contentful.com/) – Contentful's design system
 - [Contentful Field Editors](https://www.contentful.com/developers/docs/extensibility/field-editors/) – Contentful's field editor React components
 
+## Using the contentful-management SDK
+
+With the sdk and the contentful-management package installed,
+you can also create an instance of the contentful-management client using the `cmaAdapter`, which is part of the sdk, without additional configuration.
+
+
+```
+const cma = createClient(
+  { apiAdapter: sdk.cmaAdapter },
+  {
+    type: 'plain',
+    defaults: {
+      environmentId: sdk.ids.environment,
+      spaceId: sdk.ids.space,
+    },
+  }
+);
+
+cma.locale.getMany({})
+```
+
+Visit [the official docs](https://www.contentful.com/developers/docs/extensibility/app-framework/sdk/#using-the-contentful-management-api)
+to find out more.
+
+_Note:_ Feel free to uninstall the `contentful-management` package if you are not using the client.
```

Comment:
Should we even install it?

Reply:
I think having it installed doesn't hurt as it is installed with the `peerDependency` of `app-sdk` anyway and maybe we can push it with that a bit more to be used.
But would also be fine to remove it. @fabe wdyt?

Reply:
Peer Dependencies are not automatically installed.

But maybe we should make it an actual dependency of App SDK as we are relying on the CMA types.

Reply:
Nevertheless, it will be installed from somewhere when you check the node modules, even if it is not defined as a dependency.